src/Workspaces/VisualBasic/Portable/Classification/Worker.vb

```diff
@@ -137,6 +140,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Classification
             End If
         End Sub
 
+        Private Sub ClassifyShebangDirective(directiveSyntax As SyntaxNode, classificationType As String)
+            If Not _textSpan.OverlapsWith(directiveSyntax.Span) Then
+                Return
+            End If
+
+            For Each child As SyntaxNodeOrToken In directiveSyntax.DescendantTokens()
```

Comment:
Doesn't `DescendantTokens` return `SyntaxToken`s?  If so, why convert it to `SyntaxNodeOrToken` and back?